uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -551,16 +548,14 @@ sub determinePSC {
 
 =pod
 
-=head3 determineScannerID($tarchiveInfo, $to_log, $centerID, $NewScanner, $upload_id)
+=head3 determineScannerID($tarchiveInfo, $to_log, $centerID, $upload_id)
```

Comment:
Maybe indicate in the method doc that a new scanner will automatically be created if it does not exist?